pkg/tests/apis/provisioning/testdata/all-panels.json

```diff
@@ -0,0 +1 @@
+../../../../../devenv/dev-dashboards/all-panels.json
```

Comment:
is this a git simlink?  neat

any downsides to them?

Reply:
Yep! They just map to standard symlinks on the system. Only downside I know of is they don't work on Windows 8.1 and older, nor on FAT systems. So no real downside, in other words.